src/supertux/levelintro.cpp

```diff
@@ -182,6 +182,11 @@ LevelIntro::draw(Compositor& compositor)
                       Statistics::time_to_string(m_level.m_target_time), targetTimeBeaten);
     }
   }
+  py+=32;
```

Comment:
spaces around operators and in between arguments

Reply:
changed